flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/RowSerializer.java

```diff
@@ -40,6 +42,7 @@
 	private static final long serialVersionUID = 1L;
 	private final boolean[] nullMask;
 	private final TypeSerializer<Object>[] fieldSerializers;
+	private final EnumSerializer commandSerializer = new EnumSerializer(Command.class);
```

Comment:
advice: non-static final members should be initialized in constructor, otherwise make it 'static final' and use upper case variable name.